docs_user/modules/con_ceph-daemon-cardinality.adoc

```diff
@@ -7,15 +7,12 @@ colocated within the same node.
 ifeval::["{build}" != "upstream"]
 For more information, see link:https://access.redhat.com/articles/1548993[Red Hat Ceph Storage: Supported configurations].
 endif::[]
-The resulting topology depends on the available hardware, as well as the amount
-of {Ceph} services present in the Controller nodes that are going to be
-retired.
-As a general rule, the number of services that can be migrated depends on the
-number of available nodes in the cluster. The following diagrams cover the
-distribution of the {Ceph} daemons on the {Ceph} nodes where at least three
-nodes are required in a scenario that includes only RGW and RBD, without the
-{Ceph} Dashboard:
+The resulting topology depends on the available hardware, as well as the number
+of {Ceph} services in the Controller nodes that are going to be retired.
+The number of services that you can migrate depends on the number of available nodes in the cluster. The following diagrams show the distribution of {Ceph} daemons on {Ceph} nodes where at least 3 nodes are required.
```

Comment:
"The number of services that you can migrate depends on the number of available nodes in the cluster."
Is this statement kind of obvious?
I might change "number" in case it could get confused with like a serial number or something like that. 
Maybe:
"The amount of services that you can migrate is limited by the amount of available nodes in the cluster."